optuna/samplers/_cmaes.py

```diff
@@ -109,6 +131,15 @@ def __init__(
         self._logger = optuna.logging.get_logger(__name__)
         self._cma_rng = np.random.RandomState(seed)
         self._search_space = optuna.samplers.IntersectionSearchSpace()
+        self._consider_pruned_trials = consider_pruned_trials
+
+        if self._consider_pruned_trials:
+            _message = (
+                "`consider_pruned_trials = True` is experimental (supported frmo v2.0.0)."
+                "The interface can change in the future."
+            )
+            warnings.warn(_message, ExperimentalWarning)
+            _logger.warning(_message)
```

Comment:
How about using the `experimental` decorator by encapsulating this logic in a method/function and decorate that method/function?